tests/embedding/test_app.py

```diff
@@ -206,6 +205,7 @@ def change_name():
         def raise_ex():
             raise Exception("TestException")
 
+        utils.sleep(400)
```

Comment:
this isn't correct. We aren't supposed to use utils.sleep in a background thread